src/pages/evm/wallet/SendPage.vue

```diff
@@ -1,21 +1,577 @@
+                            <div v-if="token && token.price > 0" class="c-send-page__amount-fiat-footer">
+                                <div v-if="useFiat" class="c-send-page__amount-fiat" @click="toggleUseFiat">
+                                    <q-icon size="xs" name="swap_vert" class="c-send-page__amount-fiat-icon" />
+                                    <span class="c-send-page__amount-fiat-text"> {{ amountInTokens }}</span>
+                                </div>
+                                <div v-else class="c-send-page__amount-fiat" @click="toggleUseFiat">
+                                    <q-icon size="xs" name="swap_vert" class="c-send-page__amount-fiat-icon" />
+                                    <span class="c-send-page__amount-fiat-text"> {{ amountInFiat }}</span>
+                                </div>
+                                <div class="c-send-page__amount-fiat-rate">
+                                    {{ fiatRateText }}
+                                </div>
+                            </div>
+                        </template>
+                    </q-input>
+
+                    <div
+                        v-if="token && amount"
+                        :class="{
+                            'c-send-page__amount-symbol-container': true,
+                            'c-send-page__amount-symbol-container--mobile': isMobile,
+                        }"
+                    >
+                        <span class="c-send-page__amount-symbol">
+                            <span class="c-send-page__amount-transparent">{{ amount }} &nbsp;</span>
+                            <span v-if="useFiat"> {{ fiatSymbol }} </span>
+                            <span v-else> {{ token.symbol }} </span>
+                        </span>
+                    </div>
+                </div>
+            </div>
+            <div class="c-send-page__row c-send-page__row--4 row">
+                <q-space/>
+                <div class="col-auto">
+                    <div class="c-send-page__gas-fee">
+                        <div class="row c-send-page__gas-fee-title text-no-wrap">{{ $t('evm_wallet.estimated_fees') }}</div>
+                        <div class="row c-send-page__gas-fee-info">
+                            <q-space/>
+                            <div class="col-auto q-mr-xs flex flex-column items-center justify-center"><q-icon name="local_gas_station" /></div>
+                            <div class="col-auto">
+                                <div class="row text-no-wrap"> {{ gasFeeInTlos }} </div>
+                                <div class="row text-no-wrap"> {{ gasFeeInFiat }} </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="c-send-page__row c-send-page__row--5 row">
+                <div class="col">
+                    <div class="row justify-end">
+                        <q-btn
```

Comment:
should be disabled if form is invalid, you can add `:disable="!isAddressValid || !isAmountValid"`